src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ */
+contract ClaimProceeds is Controlled {
+	uint256 constant THREE_DAYS = 3 * 24 * 60 * 60;
```

Comment:
In our code, we decided to make all members private and then write getters for all of the members: https://augurproject.slack.com/archives/C031NRD9B/p1501974432517392.  (We may decide to change this policy later, though.)  This should probably be made private, but since this is a constant, I'm not sure if a getter is required here.  @MicahZoltu Are we also creating getters for all constants?